redcard.cabal

```diff
@@ -5,7 +5,7 @@ cabal-version: 1.12
 -- see: https://github.com/sol/hpack
 
 name:           redcard
-version:        0.4.0.0
+version:        0.4.0.1
```

Comment:
Strictly speaking this would be `0.4.1.0`, but it doesn't really make much difference for this package since it's published to Hackage